python/ingestion/dataset_utils.py

```diff
@@ -10,6 +10,76 @@
     RACE,
     UNKNOWN
 )
+from functools import reduce
+
+
+def melt_to_het_style_df(
```

Comment:
I love the detail in the comments because this is an entirely new concept for me.